aqueductcore/backend/services/task_executor.py

```diff
@@ -0,0 +1,13 @@
+"""Celery task execution."""
+
+from celery import Celery
+
+from aqueductcore.backend.settings import settings
+
+celery_app = Celery(
+    "tasks",
+    broker=f"amqp://{settings.rabbitmq_username}:{settings.rabbitmq_password}"
+    "@{settings.rabbitmq_host}:{settings.rabbitmq_port}",
+    backend=f"db+postgresql://{settings.postgres_username}:{settings.postgres_password}"
+    "@{settings.postgres_host}:{settings.postgres_port}/{settings.postgres_db}",
```

Comment:
Being honest, I feel this like a narrowing of configurability. E.g. you "nail" "`db+postgresql:`" and "`amqp:`" protocols. Also, instead ob having 2 parameters, you have 9. This is not a huge problems, but my preference is to have 2 connection strings configurable by user.

```suggestion
    broker=settings.celery_message_queue,
    backend=settings.celery_backend,
```

Reply:
I have added the connection string for the message queue, but for the backend, the already existing Postgres environment variables were suffice, so I kept them as before. Hopefully in future, we change the Postgres connection config into a connection string as well.

Reply:
What stops you now? =)